aiopolly/utils/strenum.py

```diff
@@ -99,7 +100,13 @@ def __prepare__(mcs, *args, sep: AnyStr = None, converter: Callable[[str], str]
         return super().__prepare__(*args, **kwargs)
 
     def __new__(mcs, cls, bases, class_dict, sep: AnyStr = None, converter: Callable[[str], str] = None):
-        mixin_type, base_enum = mcs._get_mixins_(bases)
+        # In Python 3.8 the signature of 'EnumMeta._get_mixins_' was changed from (bases) to (class_name, bases)
+        # However, 'class_name' is unneccessary and only used to print an exception
+        # So we examine the signature accordingly and pick the right one
+        if len(signature(mcs._get_mixins_).parameters) == 2:  # (class_name, bases)
```

Comment:
How about checking python version instead of signature? It would be faster than generating a signature each time and seems simpler too.
 
```suggestion
        if sys.version_info >= (3, 8):  # (class_name, bases)
```
(need also import `sys`, and delete `signature` import, if suggestion would be applied)

Reply:
This would probably work, but maybe there's a possibility they could change the signature back in a future version? Since it's a "private" method and all prefixed with `_`, I don't think there's any guarantees about its signature remaining stable.

Also in terms of speed, this is only called when the Enum classes are created right? So it would be a small hit on library import only I think.